### PR TITLE
chore: Add admin-api script for ssl-config

### PR DESCRIPTION
**Pull request by Lakshmipathi**
Signed-off-by: Lakshmipathi <lakshmi@supabase.io>

### CODE REVIEW

**Comment by Lakshmipathi** [comment]
Tested:
Following command should pass with default setup and after enforcing SSL these commands should fail.

```
$ psql "sslmode=disable sslrootcert=$HOME/Downloads/myprod-ca.crt host=db.ref.supabase.co dbname=postgres user=postgres  port=6543"
$ psql "sslmode=disable sslrootcert=$HOME/Downloads/myprod-ca.crt host=db.ref.supabase.co dbname=postgres user=postgres  port=5432"
```

**Comment by Lakshmipathi** [comment]
~Closing this PR, because [ this PR](https://github.com/supabase/supabase-admin-api/pull/103) takes care of it~ (See below)

**Comment by Lakshmipathi** [comment]
Updated commit message, this PR handles only config change and perm. not scripts.

File: ansible/files/pgbouncer_config/pgbouncer.ini.j2 [hunk 1]
```diff
@@ -361,3 +361,4 @@ default_pool_size = 15
 
 %include /etc/pgbouncer-custom/generated-optimizations.ini
 %include /etc/pgbouncer-custom/custom-overrides.ini
+%include /etc/pgbouncer-custom/ssl-config.ini
```
Comment (darora) [inline ansible/files/pgbouncer_config/pgbouncer.ini.j2:364]: would this not break the docker image, where adminapi is not included?

would suggest moving the creation of the file to `ansible/tasks/setup-pgbouncer.yml` where the other two files are currently created

**Review by darora — APPROVED** [review]

File: ansible/files/pgbouncer_config/pgbouncer.ini.j2 (hunk 1 above)
Comment (Lakshmipathi) [reply ansible/files/pgbouncer_config/pgbouncer.ini.j2:364]: okay moved this file creation from `admin-api.yml` to `setup-pgbouncer.yml`

**Pull request merged**